### PR TITLE
Fix stale readme links.

### PR DESCRIPTION
**Pull request by bryans99**

### CODE REVIEW

**Review by izzymiller — APPROVED** [review]
lgtm

File: react/typescript/access-key-demo/README.md [hunk 1]
```diff
@@ -145,11 +144,9 @@ The process above requires your local development server to be running to load t
 
 ## Related Projects
 
-- [Looker React extension template](https://github.com/looker-open-source/extension-template-react)
-- [Looker React/Redux extension template ](https://github.com/looker-open-source/extension-template-redux)
-- [Looker extension SDK](https://www.npmjs.com/package/@looker/extension-sdk)
-- [Looker extension SDK for React](https://www.npmjs.com/package/@looker/extension-sdk-react)
-- [Looker SDK](https://www.npmjs.com/package/@looker/sdk)
+- [Looker extension sdk react](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
+- [Looker extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```
Comment (jkaster) [inline react/typescript/access-key-demo/README.md:148]:
```suggestion
- [Looker Extension SDK React](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
- [Looker Extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```

Feel free to ignore

File: react/typescript/helloworld-ts/README.md [hunk 2]
```diff
@@ -109,11 +109,9 @@ The process above describes how to run the extension for development. Once you'r
 
 ## Related Projects
 
-- [Looker React extension template](https://github.com/looker-open-source/extension-template-react)
-- [Looker React/Redux extension template ](https://github.com/looker-open-source/extension-template-redux)
-- [Looker extension SDK](https://www.npmjs.com/package/@looker/extension-sdk)
-- [Looker extension SDK for React](https://www.npmjs.com/package/@looker/extension-sdk-react)
-- [Looker SDK](https://www.npmjs.com/package/@looker/sdk)
+- [Looker extension sdk react](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
+- [Looker extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```
Comment (jkaster) [inline react/typescript/helloworld-ts/README.md:113]:
```suggestion
- [Looker Extension SDK React](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
- [Looker Extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```

File: react/typescript/kitchensink/README.md [hunk 3]
```diff
@@ -464,11 +464,9 @@ Note that the additional JavaScript files generated during the production build
 
 ## Related Projects
 
-- [Looker React extension template](https://github.com/looker-open-source/extension-template-react)
-- [Looker React/Redux extension template ](https://github.com/looker-open-source/extension-template-redux)
-- [Looker extension SDK](https://www.npmjs.com/package/@looker/extension-sdk)
-- [Looker extension SDK for React](https://www.npmjs.com/package/@looker/extension-sdk-react)
-- [Looker SDK](https://www.npmjs.com/package/@looker/sdk)
+- [Looker extension sdk react](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
+- [Looker extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```
Comment (jkaster) [inline react/typescript/kitchensink/README.md:468]:
```suggestion
- [Looker Extension SDK React](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
- [Looker Extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```

File: react/typescript/looks-query-redux/README.md [hunk 4]
```diff
@@ -104,9 +105,9 @@ The process above requires your local development server to be running to load t
 
 ## Related Projects
 
-- [Looker React extension template](https://github.com/looker-open-source/extension-template-react)
-- [Looker kitchensink extension template ](https://github.com/looker-open-source/extension-template-kitchensink)
-- [Looker extension SDK for React](https://www.npmjs.com/package/@looker/extension-sdk-react)
-- [Looker SDK](https://www.npmjs.com/package/@looker/sdk)
+- [Looker extension sdk react](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
+- [Looker extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```
Comment (jkaster) [inline react/typescript/looks-query-redux/README.md:109]:
```suggestion
- [Looker Extension SDK React](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
- [Looker Extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```

File: react/typescript/looks-query/README.md [hunk 5]
```diff
@@ -104,9 +105,9 @@ The process above requires your local development server to be running to load t
 
 ## Related Projects
 
-- [Looker React extension template](https://github.com/looker-open-source/extension-template-react)
-- [Looker kitchensink extension template ](https://github.com/looker-open-source/extension-template-kitchensink)
-- [Looker extension SDK for React](https://www.npmjs.com/package/@looker/extension-sdk-react)
-- [Looker SDK](https://www.npmjs.com/package/@looker/sdk)
+- [Looker extension sdk react](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
+- [Looker extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```
Comment (jkaster) [inline react/typescript/looks-query/README.md:109]:
```suggestion
- [Looker Extension SDK React](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk-react)
- [Looker Extension SDK](https://github.com/looker-open-source/sdk-codegen/tree/main/packages/extension-sdk)
```

**Review by jkaster — APPROVED** [review]
Looks good. Feel free to take the suggestions or not.

**Pull request merged**